### DISCUSSION TITLE
psycopg2.errors.UndefinedColumn: column dag.concurrency does not exist

### DISCUSSION THREAD

**Discussion by mk12306** [apache/airflow#25284] [General]
### Apache Airflow version

2.1.4

### What happened

psycopg2.errors.UndefinedColumn: column dag.concurrency does not exist

### What you think should happen instead

enter the systerm

### Operating System

CentOS Linux 7 (Core)

### Deployment

Docker-Compose

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
You likely did not upgrade your database. Please refer to airlfow Upgrading docs.
